docs/source/backup/index.rst

```diff
@@ -19,6 +19,9 @@ Backup
 
 Using :ref:`sctool backup <sctool-backup>` command, you can schedule a backup of a managed cluster.
 Backups and repairs are scheduled in the same manner, you can start, stop, resume, and track task progress on demand.
+
+.. note:: If you are using ScyllaDB Manager deployed by ScyllaDB Operator, see dedicated `ScyllaDB Operator documentation <https://operator.docs.scylladb.com/stable/manager.html>`_.
```

Comment:
FYI, you can use [RST. Include](https://docutils.sourceforge.io/docs/ref/rst/directives.html#including-an-external-document-fragment) or repeat sections. It's not required here.